frontend/src/component/changeRequest/ChangeRequestStatusBadge/ChangeRequestStatusBadge.tsx

```diff
@@ -16,8 +22,12 @@ const ReviewRequiredBadge: VFC = () => (
 const DraftBadge: VFC = () => <Badge color='warning'>Draft</Badge>;
 
 export const ChangeRequestStatusBadge: VFC<IChangeRequestStatusBadgeProps> = ({
-    state,
+    changeRequest,
 }) => {
+    if (!changeRequest) {
```

Comment:
When changeRequest can be null? If it can be null, should be stated in type.

Reply:
When loading - sure I will update the type